### PR TITLE
Enhance style of code blocks in MarkBind

### LINKED ISSUE #347: Make code blocks stand out
v1.9.0
Current: Code blocks blend in with the background
image

Suggested: Make them stand out a bit, but as subtly as possible. For reference, it was like this before:
image

### PR DESCRIPTION
**Pull request by Chng-Zhi-Xuan**
**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [x] Enhancement to an existing feature

Resolves #347 

**What is the rationale for this request?**
Current code block styles blend in too much. 

**What changes did you make? (Give an overview)**
- Change CSS specific to code blocks

Before |
--- |
347-before

After |
--- |
347-after

**Is there anything you'd like reviewers to focus on?**
- New style

**Testing instructions:**
- Use `Netlify Preview`

### CODE REVIEW

**Comment by damithc** [comment]
Is there an example of bock quotes in the netlify preview?

**Comment by damithc** [comment]
> Is there an example of bock quotes in the netlify preview?

Sorry, this question is not applicable here. I was thinking of quoted text, not code blocks.

**Comment by damithc** [comment]
Make the border as subtle as possible. Or use slightly grey background instead of a border? I'm keen to minimize lines as we seem to end up with 'too many lines too close together' when using nested panels :-p

image

**Comment by Chng-Zhi-Xuan** [comment]
**Update**
- Changed border style to 

How about this ? |
--- |
347-after-v2

I feel the lines are a good indicator on the amount of levels nested.

**Comment by damithc** [comment]
> Changed border style. Use Netlify Preview to get a better look.

This one looks good. Let's go ahead with this.

File: asset/css/markbind.css [hunk 1]
```diff
@@ -6,6 +6,15 @@ code {
     word-break: normal;
 }
 
+pre > code.hljs {
+    background: ghostwhite;
+    border: 1px solid rgb(200, 200, 200);
+    border: 1px solid rgba(200, 200, 200, 0.3);
+    border-radius: 5px;
+    -webkit-background-clip: padding-box; /* for Safari */
+    background-clip: padding-box; /* for IE9+, Firefox 4+, Opera, Chrome */
```
Comment (yamgent) [inline asset/css/markbind.css]: Just need to move this to the correct alphabetical order and we are good to go.

**Comment by Chng-Zhi-Xuan** [comment]
**Update**
- Squashed changes requested

**Pull request merged**